embedded-pulsar/src/main/java/com/playtika/test/pulsar/EmbeddedPulsarBootstrapConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package com.playtika.test.pulsar;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.testcontainers.containers.PulsarContainer;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Slf4j
+@Configuration
+@ConditionalOnExpression("${embedded.containers.enabled:true}")
+@ConditionalOnProperty(name = "embedded.pulsar.enabled", matchIfMissing = true)
+@EnableConfigurationProperties(PulsarProperties.class)
+public class EmbeddedPulsarBootstrapConfiguration {
+
+    @Bean(name = PulsarProperties.EMBEDDED_PULSAR)
+    public PulsarContainer embeddedPulsar(final PulsarProperties pulsarProperties,
+                                          final ConfigurableEnvironment environment) {
+        PulsarContainer pulsarContainer = new PulsarContainer(pulsarProperties.imageTag);
```

Comment:
any possibilities to configure pulsar container?

Reply:
Pulsar container does not require any user,password. The only thing that could be added is to create default topic - but if topic does not exist it will be automatically created when consumer/producer for this topic is created.